pennylane/concurrency/executors/native/conc_futures.py

```diff
@@ -44,7 +45,10 @@ class ProcPoolExec(PyNativeExec):
 
     @classmethod
     def _exec_backend(cls):
-        return ProcessPoolExecutor
+        ctx = multiprocessing.get_context("spawn")
+        return lambda max_workers=None, **kwargs: ProcessPoolExecutor(
+            max_workers=max_workers, mp_context=ctx, **kwargs
+        )
```

Comment:
```suggestion
        return partial(ProcessPoolExecutor, mp_context=get_context("spawn"))
```

The use of a partial here would be clearer than a lambda